addons/sales_team/models/crm_team.py

```diff
@@ -298,6 +221,18 @@ def write(self, values):
             self._add_members_to_favorites()
         return res
 
+    @api.multi
+    def unlink(self):
```

Comment:
I would remove this override. Some people are working on a way to mark some data as master data. Anyway Odoo should still work without those teams data even if it means that some default values are not computed.

Reply:
After discussion, we keep it  because if you remove some data and then install website sale it crashes. We keep this code until a better solution to manage sales team is fonhd.